### PR TITLE
Task/frontend41 create video chat component

### PR DESCRIPTION
**Pull request by droztoff**

### CODE REVIEW

**Comment by droztoff** [comment]
[Implement video chat component](https://trello.com/c/jSDnMYJT/41-implement-video-chat-component)

File: shared/src/common/types/comment/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export type Comment = {
+  id: number;
+  avatar: string;
+  name: string;
+  date_added: string;
```
Comment (kyrylo-burmelov) [inline shared/src/common/types/comment/index.ts]: use camel case. `date_added`

File: frontend/src/components/video-chat/video-comment/video-comment.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { FC } from 'common/types/types';
+import { Comment } from 'shared/src/common/types/comment';
+
+import './video-comment.scss';
```
Comment (kyrylo-burmelov) [inline frontend/src/components/video-chat/video-comment/video-comment.tsx]: use module scss

File: frontend/src/components/video-chat/video-comment/video-comment.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+.video-comment .main-part-comment .commentators-avatar {
+  width: 34px;
+  height: 34px;
+  margin-right: 12px;
+  background-size: cover;
+  border-radius: 50%;
+}
+
+.video-comment .main-part-comment .commentators-name {
+  color: #ffffff;
```
Comment (kyrylo-burmelov) [inline frontend/src/components/video-chat/video-comment/video-comment.scss]: all colors must be variables

File: frontend/src/components/common/send-message/send-message.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+              fill="#363C3A"
+            />
+          </svg>
```
Comment (kyrylo-burmelov) [inline frontend/src/components/common/send-message/send-message.tsx]: move svg's to icon component

File: frontend/src/assets/css/styles.scss [hunk 5]
```diff
@@ -1 +1,14 @@
...
+  justify-content: space-between;
+}
+
+.video-chat-component {
+  box-sizing: border-box;
+  width: 500px;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+}
```
Comment (kyrylo-burmelov) [inline frontend/src/assets/css/styles.scss]: why it's global styles?

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/send-message/send-message.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        <button className={styles['choose-emoji']}>
+          <Icon name={IconName.EMOJI} />
+        </button>
+        <button className={styles['send-message']}>
+          <Icon name={IconName.SENDMESSAGE} />
+        </button>
+      </div>
+    </div>
+  );
+};
```
Comment (kyrylo-burmelov) [inline frontend/src/components/common/send-message/send-message.tsx]: - each input must have controlled state (can be in a component)
- each button must have on click handler in a container (you can leave action empty, no need to add redux state for now)

File: frontend/src/components/video-chat/video-comment/video-comment.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  comment: Comment;
+};
+
+const VideoComment: FC<Props> = ({ comment }) => {
+  return (
+    <div className={styles['video-comment']}>
+      <div className={styles['main-part-comment']}>
+        <img className={styles['commentators-avatar']} src={comment.avatar} alt={comment.name} />
+        <p className={styles['commentators-name']}>{comment.name}</p>
+        <span className={styles['dispatch-time']}>{comment.dateAdded}</span>
```
Comment (kyrylo-burmelov) [inline frontend/src/components/video-chat/video-comment/video-comment.tsx]: you would probably have date timestamp from backend. Use dayjs to prepare date formatting https://day.js.org/

File: frontend/src/components/video-chat/video-comment/video-comment.tsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+type Props = {
+  comment: Comment;
+};
+
+const VideoComment: FC<Props> = ({ comment }) => {
+  return (
+    <div className={styles['video-comment']}>
+      <div className={styles['main-part-comment']}>
+        <img className={styles['commentators-avatar']} src={comment.avatar} alt={comment.name} />
```
Comment (kyrylo-burmelov) [inline frontend/src/components/video-chat/video-comment/video-comment.tsx]: Having in mind that user avatar is not required field. So some users do not have avatar. You need to handle that and have default icon when avatar does not exist

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]

**Review by kyrylo-burmelov — APPROVED** [review]

**Pull request merged**